pkg/repos/runtimes/python/python.go

```diff
@@ -134,6 +134,9 @@ func (r *Runtime) Setup(ctx context.Context, dataRoot, toolSource string, env []
 	}
 
 	newEnv := runtimeEnv.AppendPath(env, venvBinPath)
+	if runtime.GOOS == "windows" && os.Getenv("PYTHONIOENCODING") != "" {
```

Comment:
Ah, crap. I didn't catch this before, but this should be `== ""`

Reply:
Oh duh. Pair programming made us worse haha.